pkgs/tools/admin/aws-auth/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, makeWrapper, jq, awscli }:
+
+stdenv.mkDerivation rec {
+  version = "unstable-2017-07-24";
```

Comment:
This is not a valid version since it doesn't start with a digit. So this package will appear to be named `aws-auth-unstable` to nix-env.